src/main/java/io/confluent/idesidecar/restapi/connections/DirectConnectionState.java

```diff
@@ -41,6 +43,20 @@ public class DirectConnectionState extends ConnectionState {
           .orElse(5L)
   );
 
+  static final ConnectionStatus INITIAL_STATUS = new ConnectionStatus(
+      null,
+      new KafkaClusterStatus(
+          ConnectedState.ATTEMPTING,
+          null,
+          null
+      ),
+      new SchemaRegistryStatus(
+          ConnectedState.ATTEMPTING,
+          null,
+          null
+      )
+  );
```

Comment:
Using `ConnectedState.ATTEMPTING` as the initial state of Direct connections is fine for now, given that the endpoints `GET /gateway/v1/connections` and  `GET /gateway/v1/connections/:id` call `DirectConnectionState#getKafkaConnectionStatus()`, which synchronously checks the status of the connection. 

With the availability of Direct connections, I expect users to manage more than one connection. As a consequence, I think it's no longer feasible to perform the status check synchronously when calling the endpoint `GET /gateway/v1/connections`. So, we probably want to migrate to asynchronously checking the status of Direct connections (if not of all connection types) sooner rather than later, in which case we should use `ConnectedState.NONE` as the initial state and set the state to `ConnectedState.ATTEMPTING` when attempting to establish a connection to the Kafka cluster. WDYT? I could capture it in an issue if you think it's worth addressing it.

Reply:
Great suggestion. And thanks for capturing an issue, when you get a chance.

Reply:
Thanks! I created https://github.com/confluentinc/ide-sidecar/issues/181.